recipes-kernel/linux/files/reTerminal-overlay.dts

```diff
@@ -0,0 +1,250 @@
+/*
```

Comment:
It's not a good idea to use the static dts code here.
Because the code maybe changed in the future.
Every time we compile the repo, we get the latest dts code in [here](https://github.com/Seeed-Studio/meta-seeed-cm4/blob/57d1b68d73e625fe6a4cb14372a0cb7c42bae9c5/recipes-kernel/linux/linux-raspberrypi_5.10.bbappend#L13)

Reply:
Sorry, but I strongly disagree here. Dynamically fetching breaks a number good practises in a Yocto build. Let me explain:
- first and foremost, the URL failed for me. I don't know what the deeper reason was, but it also doesn't matter. The recipe failed with being unable to get the DTS from the source, probably because Github changed something in its protocols again, and then you can't build any more. This is bad.
- second, this breaks reproducibility. A user who gets a layer at a specific git revision expects it to behave identical at all points in time. So if you change the DTS at will, this is not given any more.
- third results from second: if you carry the DTS in the recipe, or alternatively provide an own kernel tree, then all updates become visible as version bumps in. the git log of the layer. This is expected behaviour so users can actually bisect build failures.

There are a number of similar issues left in the layer that I intend to fix in my fork in the future.

Greetz,
Josef

Reply:
Thank you for your reply.

- First of all, the main purpose that we create this repo is we want to use the latest code any time when we use it.
if you have problem with using it, you can create issue here, and I'm glad to help you solve them.
- Secondly, let me take the reTerminal-overlay.dts for example.if we use the static code in this repo. Everytime I update 
the code of reTerminal-overlay.dts in it's origin repo. I need to sync the code in this repo manually.This takes a big heavy
of our maintainning work.
- Finally, if you have problem using the reTerminal-overlay.dts, you can alse check it's update log in the origin [repo](https://github.com/Seeed-Studio/seeed-linux-dtoverlays).

Thank you anyway.

Reply:
I see. For the outlined reasons I absolutely can not agree. If the purpose of this repository is to "just have an internal testing repository ", for which you are intentionally trading stability and usability versus ease of maintenance, then I strongly suggest to add a note the reader.

Please be clear about the layer not being meant nor ready for production use.

Reply:
> I see. For the outlined reasons I absolutely can not agree. If the purpose of this repository is to "just have an internal testing repository ", for which you are intentionally trading stability and usability versus ease of maintenance, then I strongly suggest to add a note the readme.
> 
> Please be clear about the layer not being meant nor ready for production use.

Thank you for your suggestion. I will add the notice into the notice.